### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by yamensaban** [delta-io/delta#1993]
#### Which Delta project/connector is this regarding?

- [X] Spark
- [ ] Standalone
- [ ] Flink
- [ ] Kernel
- [ ] Other (fill in here)

#### Steps to reproduce

`from delta import *

data = spark.range(0, 5)
data.write.format("delta").saveAsTable("deltatable")`

#### Observed results

The table gets created with the wrong Input/Output Format classes _Sequence_, so when I try to query the table from Hive I get an error.
When I use save() method as below, and then create an External table in Hive it works fine.
`data.write.format("delta").save("/tmp/deltatest/deltatable")`

### Environment information

* Delta Lake version: 2.3
* Spark version: 3.3.1
* Scala version: 2.12

**Comment by vkorukanti** [comment]
cc. @tdas

**Comment by vkorukanti** [comment]
@yamensaban You can't use the entry created by Spark in HMS to query the table from Hive. Hive needs another entry in HMS and a separate jar in order to read the Delta table. [Here](https://github.com/delta-io/connectors/tree/master/hive) are the instructions.

**Comment by yamensaban** [comment]
@vkorukanti Yes I know, I have already added the following properties to hive-site.xml under spark and hiveserver2 
![image](https://github.com/delta-io/delta/assets/74052115/878326af-c0f4-4e17-aeee-82987a34c608)

I have also added `delta-hive-assembly_2.12-0.6.0.jar` file under hive/lib and spark/jars path, External tables created after the data is written by Spark using save() method works fine but when it comes to saveAsTable() method the table gets created as below, with wrong input format and cannot be quired
![image](https://github.com/delta-io/delta/assets/74052115/11b3c309-9487-43ba-8a8d-b303ebb501d0)

When I try to query the table from Spark SQL it works fine
![image](https://github.com/delta-io/delta/assets/74052115/3a26b8de-a02f-49fc-8884-caa1b649b3f6)

**Comment by orthoxerox** [comment]
@yamensaban It's a known limitation of the connector. You have to create a separate external table definition *from Hive* that points to the same location to read it.

**Comment by foss-contributor** [comment]
@yamensaban, thank you for your contribution! This issue has been inactive for over 18 months and will be closed in one week. To keep it open, please leave a comment on this issue or contact us in the `#contributors` channel on [Delta Lake Slack](https://go.delta.io/slack). We appreciate your involvement in Delta Lake!
